h2o-algos/src/main/java/hex/rulefit/RuleFit.java

```diff
@@ -441,12 +442,15 @@ int getBestLambdaIndexCornerCase(double[] deviance, double[] lambdas) {
                     rule = ruleEnsemble.getRuleByVarName(getVarName(entry.getKey(), classNames));
                 } else {
                     rule = new Rule(null, entry.getValue(), entry.getKey());
+                    // linear rule applies to all the rows
+                    rule.support = 1.0;
                 }
                 rule.setCoefficient(entry.getValue());
                 rules.add(rule);
             }
-            Comparator<Rule> ruleAbsCoefficientComparator = Comparator.comparingDouble(Rule::getAbsCoefficient).reversed();
-            rules.sort(ruleAbsCoefficientComparator);
+//            todo:  this is not needed here now, but will be when deduplication will be made before glm (mojo change needed)            
```

Comment:
todo?

Reply:
will remove, this is not related to this PR

Reply:
8b1699f